.github/workflows/ci.yml

```
@@ -96,6 +97,36 @@ jobs:
       if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')
       env:
         GITHUB_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+  alpine-release:
+    runs-on: ubuntu-latest
+    container: node:15-alpine3.12
+    env:
+      MAKEFLAGS: -j4
+    steps:
+      - uses: actions/checkout@v2
+      - name: install build deps
```

:art: let's keep these consistent

```suggestion
      - name: Install additional dependencies
```

---

Done